packages/web-ui/src/sections/Document/Sidebar/Files/useTree/index.test.ts

```diff
@@ -47,6 +47,49 @@ describe('useTree', () => {
     expect(root.children).toHaveLength(5)
   })
 
+  it('renders folder with same name as file in the same level', async () => {
+    const { result } = renderHook(() =>
+      useTree({
+        documents: [
+          { path: 'a', documentUuid: '1' },
+          { path: 'a/b', documentUuid: '2' },
```

Comment:
This was the bug. When a folder and file was named the same the tree was not rendering the folder `a` and the file `b`

Reply:
nice